source/_patterns/00-atoms/04-colors/02-brand-colors.md

```diff
@@ -0,0 +1,4 @@
+---
+title: Brand-Colors
+state: completed
```

Comment:
"complete" without the "d".
<img width="118" alt="Screenshot 2019-07-15 at 14 37 02" src="https://user-images.githubusercontent.com/4340405/61216612-0eba9380-a70e-11e9-99ca-333a5f78d32d.png">
The dot is grey, when it does not receive a known state. But it will be green, when it is "complete". :)

Reply:
Btw, please remove the "color" atom and its code.